packages/conf-opencc1_1/conf-opencc1_1.1/opam

```diff
@@ -10,17 +10,25 @@ tags: [
 ]
 
 build: [
-  ["sh" "-exec" "cc $CFLAGS test.c -l:libopencc.so.1.1"]
```

Comment:
No opencc package provides PKG config configuration on linux.
I prefer to keep the old version here, even if it fails. People will have to set the CFLAGS manually as they had to do in the past